src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/CaseAssignmentControllerTest.java

```diff
@@ -52,18 +52,18 @@ void givenIncorrectReference_whenValidateCaseAndPin_shouldReturnUnauthorised() {
     void givenIncorrectPin_whenValidateCaseAndPin_shouldReturnUnauthorised() {
         givenCaseIsFound();
         doThrow(new PinNotMatchException()).when(defendantPinToPostLRspecService).validatePin(
-            any(CaseData.class),
+            any(CaseDetails.class),
             anyString()
         );
 
         doPost("", "123", VALIDATE_PIN_URL, "123")
```

Comment:
in this case we should test that it returns bad request

Reply:
Changes have been made in the CaseAssignmentController

Reply:
i understand but we need to make changes in the test. So we need to change the title and what status we expect. In the title of the test we are expecting it to return unauthorised. but we should expect it to return 400

Reply:
cool can see your changes in the correct test but need to change the titele of the test _shouldReturnBadRequest